packages/react-devtools-shared/src/devtools/views/Components/Tree.js

```diff
@@ -103,6 +104,7 @@ export default function Tree(props: Props) {
     function handleStopInspectingNative(didSelectNode) {
       if (didSelectNode && focusTargetRef.current !== null) {
         focusTargetRef.current.focus();
+        logEvent({event_name: 'inspect-element-success'});
```

Comment:
Should this also just be `select-element` since it ends up being what it is?

Reply:
good catch, this should be `select-element` with `source: 'inspect-element'`